src/object/circleplatform.cpp

```diff
@@ -0,0 +1,84 @@
+//  Copyright (C) 2020 Daniel Ward <weluvgoatz@gmail.com>
```

Comment:
My editor shows, that this file has mixed indentation (tabs and spaces) please change all to spaces.